public/stylesheets/login.css

```
@@ -43,6 +43,11 @@
   display: none;
 }
 
+#login-page .login-error-message {
+  margin-top: 1rem;
+  color: #ca3c3c;
```

```suggestion
  color: 'red'
```

모든 에러메시지들은 빨간색(`red`)을 사용하고 있는데

통일성을 위해 동일한 색상을 사용하는 것이 어떨까요?